src/radical/pilot/agent/executing/base.py

```diff
@@ -328,7 +329,8 @@ def _create_exec_script(self, launcher, task):
 
             tmp += self._separator
             tmp += '# output file detection (i)\n'
-            tmp += "ls | sort | grep -ve '^%s\\.' > %s.files\n" % (tid, tid)
+            tmp += "ls | sort | grep -ve '^%s\\.' > %s/%s.files\n" \
+                    % (env_sbox, tid, tid)
```

Comment:
here order is also got mixed, should be `% (tid, env_sbox, tid)`

Reply:
Thanks!